### ISSUE TITLE
package.json references assemble-toc instead of this repository

### ISSUE THREAD

**Issue by jt000** [jonschlinkert/grunt-toc#1]
I think the package.json was incorrectly cut\pasted from another repository. Took me a while to find out where the source from NPM actually existed... https://www.npmjs.com/package/grunt-toc

**Comment by jonschlinkert** [comment]
indeed, I think I created this back before the conveniences of project generators lol. thanks for the heads up!
